src/model/bpmn/internal/shape/kinds.ts

```diff
@@ -132,7 +132,7 @@ export enum ShapeBpmnMarkerKind {
  * @category BPMN
  */
 export enum ShapeBpmnSubProcessKind {
-  AD_HOC = 'ad_hoc',
+  AD_HOC = 'adHoc',
```

Comment:
**thought:** value could be set to `adhoc` without impact if we prefer to only use lower-case values.